dashboard/modules/job/job_head.py

```diff
@@ -261,7 +265,10 @@ async def upload_package(self, req: Request):
         )
         logger.info(f"Uploading package {package_uri} to the GCS.")
         try:
-            upload_package_to_gcs(package_uri, await req.read())
+            data = await req.read()
+            await self._upload_package_thread_pool_executor.submit(
+                upload_package_to_gcs, package_uri, data
```

Comment:
I saw it's just gcs kv put. Is it easy to just implement an async version of it using gcs_aio_client?

Reply:
I tried that, but `upload_package_to_gcs` is also used by CLI (Job SDK) code in sync form so it's non trivial to change.